solutions/src/test/java/ru/sberbank/school/task09/RouteServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package ru.sberbank.school.task09;
+
+import org.junit.jupiter.api.*;
+
+import java.io.File;
+
+public class RouteServiceTest {
+    private final static String filePath =
+            new File("src\\test\\java\\ru\\sberbank\\school\\task09").getAbsolutePath();
+
+    private final static String cityOne = "London";
+    private final static String cityTwo = "Moscow";
```

Comment:
&#39;static&#39; modifier out of order with the JLS suggestions.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1156270221>*